src/components/Header.js

```diff
@@ -11,10 +11,13 @@ export default function Header({ children }) {
 
 const HeaderStyled = styled.h2`
   display: grid;
+  height: 80px;
```

Comment:
Be careful setting fixed height. If the parent font-size increases, you would face an overflow:

![image](https://user-images.githubusercontent.com/10058950/122027466-6dda2480-cdcb-11eb-8bc5-8dfb554b7d0a.png)